tests_ovs_extended/feature_ovs_test.go

```diff
@@ -40,8 +42,16 @@ func (s *kindFeatOvsSuite) BeforeTest(suiteName, testName string) {
 
 func TestRunFeatureOvsSuite(t *testing.T) {
 	if !*smartVFFlag {
-		parallel.Run(t, new(kindFeatOvsSuite), "TestScale_from_zero", "TestNse_composition", "TestSelect_forwarder")
+		featureOvsSuite := new(kindFeatOvsSuite)
+		parallel.Run(t, featureOvsSuite,
+			parallel.WithRunningTestsSynchronously(
+				featureOvsSuite.TestScale_from_zero,
+				featureOvsSuite.TestSelect_forwarder))
 	} else {
-		parallel.Run(t, new(features_ovs.Suite), "TestScale_from_zero", "TestNse_composition", "TestSelect_forwarder")
+		featureOvsSuite := new(features_ovs.Suite)
+		parallel.Run(t, featureOvsSuite,
+			parallel.WithRunningTestsSynchronously(
+				featureOvsSuite.TestScale_from_zero,
+				featureOvsSuite.TestSelect_forwarder))
```

Comment:
Looks like TestNse_composition is missed

Reply:
it looks like he's gone